cmd/run.go

```diff
@@ -46,6 +47,7 @@ This will reconcile any events for the Combination and Template resources.
 			return err
 		}
 
+		rootLog.Info("starting combo", "version", version.ComboVersion)
```

Comment:
Should we also include the `KubernetesVersion` and `GitCommit`?

Reply:
It looks like there's already an existing String method that could dump out that information.

Reply:
What is that method, just out of curiosity?

Reply:
https://github.com/operator-framework/combo/blob/main/pkg/version/version.go#L14-L18 is the method I had in mind. It just outputs a formatted string that contains the version information we pass when building the combo binary.

Reply:
I guess the logger takes in raw strings or doesn't process formatting characters for some other reason because when  passed in it results in a message with `\n` being printed explicitly instead of actual newlines (see here `{"level":"info","ts":1649706102.7005851,"msg":"Starting Combo","versions:":"Combo version: v0.0.1-4-g83a20a6\nGit commit: 83a20a646ce3f6562aa47ed90b16e3df81cbaf5e\nKubernetes version: v0.22.2"}`. Should I try and just manually input the version information for output?

Reply:
Yep - let's just revert back to using the version.ComboVersion field directly.